### PR TITLE
Fix W18-51 to -65 abstracts (fixes #714)

### LINKED ISSUE #714: Many abstracts in W18-51 to W18-65 are incorrect
#713 fixes one abstract; the others also need to be fixed.

I haven't checked other workshops from the same conference, but those should probably be spot-checked too.

### PR DESCRIPTION
**Pull request by GuyAglionby**
Semi-automatically done -- used [this script](https://gist.github.com/GuyAglionby/a9d184ddec0253fe8d49c49acd88bbe5) then manually verified the changes (though there were a lot so it's possible I missed a couple of extraction errors). 

It looks like there were two classes of changes: human-made (from minor edits to more serious reworks) and errors (e.g. text cut off mid sentence). Hope this isn't too large a change!

### CODE REVIEW

**Comment by davidweichiang** [comment]
Thanks! I've never used pdfquery; how does it know where to stop extracting lines for the abstract?

File: data/xml/W18.xml [hunk 1]
```diff
@@ -12115,6 +12186,7 @@
       <author><first>Alexander</first><last>Fraser</last></author>
       <pages>648–654</pages>
       <url>W18-6446</url>
+      <abstract>We present the LMU Munich machine translation systems for the English–German language pair. We have built neural machine translation systems for both translation directions (English→German and German→English) and for two different domains (the biomedical domain and the news domain). The systems were used for our participation in the WMT18 biomedical translation task and in the shared task on machine translation of news.1,2 The main focus of our recent system development efforts has been on achieving improvements in the biomedical domain over last year’s strong biomedical translation engine for English→German (Huck et al., 2017a). Considerable progress has been made in the latter task, which we report on in this paper.</abstract>
```
Comment (davidweichiang) [inline data/xml/W18.xml]: delete `1,2`

File: data/xml/W18.xml [hunk 2]
```diff
@@ -12463,6 +12572,7 @@
       <author><first>Stelios</first><last>Piperidis</last></author>
       <pages>928–933</pages>
       <url>W18-6484</url>
+      <abstract>This paper describes the submission of the Institute for Language and Speech Processing/Athena Research and Innovation Center (ILSP/ARC) for the WMT 2018 Parallel Corpus Filtering shared task. We explore several properties of sentences and sentence pairs that our system explored in the context of the task with the purpose of clustering sentence pairs according to their appropriateness in training T systems. We also discuss alternative methods for ranking the sentence pairs of the most appropriate clusters with the aim of generating the two datasets (of 10 and 100 million words as required in the task) that were evaluated. By summarizing the results of several experiments that were carried out by the organizers during the evaluation phase, our submission achieved an average BLEU score of 26.41, even though it does not make use of any language-specific resources like bilingual lexica, monolingual corpora, or MT output, while the average score of the best participant system was 27.91.</abstract>
```
Comment (davidweichiang) [inline data/xml/W18.xml]: T -> MT

File: data/xml/W18.xml [hunk 3]
```diff
@@ -12905,7 +13020,7 @@
       <author><first>Marilyn</first><last>Walker</last></author>
       <pages>296–306</pages>
       <url>W18-6536</url>
-      <abstract>Question Generation is the task of automatically creating questions</abstract>
+      <abstract>Question Generation is the task of automatically creating questions from textual input. In this work we present a new Attentional Encoder–Decoder Recurrent Neural Network model for automatic question generation. Our model incorporates linguistic features and an additional sentence embedding to capture meaning at both sentence and word levels. The linguistic features are designed to capture information related to named entity recognition, word case, and entity coreference resolution. In addition our model uses a copying mechanism and a special answer signal that enables generation of numerous diverse questions on a given sentence. Our model achieves state of the art results of 19.98 Bleu 4 on a benchmark Question Generation dataset, outperforming all previously published results by a significant margin. A human evaluation also shows that the added features improve the quality of the generated questions.</abstract>
```
Comment (davidweichiang) [inline data/xml/W18.xml]: `Bleu 4` -> `Bleu_4`

**Comment by GuyAglionby** [comment]
Thanks for the review and taking the time to find examples of errors! I have rewritten the scraping code to be both able to extract formatting and more robust in retrieving all characters in the document. The resulting changes are mostly for formatting, but also some short phrases that the original code missed.

I think most of the errors you noticed were because of slight misalignment on the y axis of some characters, which led to pdfquery misordering or ignoring them. I've rewritten this code to deal with a wider range of edge cases ([here](https://gist.github.com/GuyAglionby/4b55d00803710f2e2e9877fd18b5a491) if interested).

To your first question, it turns out that the abstract itself is all contained within a single xml element, so I just iterate through all of the characters in that ([this xpath](https://gist.github.com/GuyAglionby/4b55d00803710f2e2e9877fd18b5a491#file-fix-abstracts-py-L224)). For some papers the first header is included in that element; this is "Introduction" 99% of the time so I filter that out.

**Comment by davidweichiang** [comment]
Thank you! I can’t get this until mid-January, but maybe someone else will be able to.

**Comment by GuyAglionby** [comment]
Thanks! Will do

**Pull request merged**